test-report/action.yml

```diff
@@ -64,10 +64,40 @@ inputs:
         - step-summary
     default: 'checks'
     required: false
+  github-token:
+    description: >-
+      GitHub token used to access the GitHub API. Defaults to secrets.GITHUB_TOKEN.
+    default: ${{ github.token }}
+    required: false
 runs:
   using: "composite"
   steps:
+    - name: Check artifact existence via GitHub API
```

Comment:
Wonder whether this could be done in `phoenix-actions/test-reporting` instead, so we don't need to maintain the logic in our end, WDYT?

Reply:
We evaluated `dorny/test-reporter` a long time ago, and we chose `phoenix-actions/test-reporting`, I do not remember why.  It seems that `phoenix-actions/test-reporting` has not had releases for a year or so, while `dorny/test-reporter` has had recent releases. Both actions are clones, so I think there is no issue with changing it.